### PR TITLE
Implement some methods for `Gentoo::Base::Package`

### PR DESCRIPTION
**Pull request by hirocaster**
I manage gentoo linux since few days ago by mitamae. (Thank good product👍 )
So I need this.

This PR checked in my gentoo linux.

ref: https://github.com/mizzy/specinfra/commit/d68c3c59616ba43fb4b9798666b98c6ec92f8bcc

### CODE REVIEW

**Comment by k0kubun** [comment]
Thanks :+1:

**Comment by k0kubun** [comment]
Released mitamae as v1.3.4.

**Comment by sorah** [comment]
Personally speaking, Portage has a lot of different concept from _usual_ package management systems. So you may have interested in https://github.com/sorah/itamae-plugin-resource-portage too.

File: mrblib/specinfra/command/gentoo/base/package.rb [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+
+    def get_version(package, opts=nil)
+      "equery -q list #{package} | sed -e 's!^.*/?#{package}-!!'"
+    end
+
+    def install(package, version=nil, option='')
+      if version
+        full_package = "=#{package}-#{version}"
+      else
+        full_package = package
```
Comment (sorah) [inline mrblib/specinfra/command/gentoo/base/package.rb:15]: FYI, this is called _atom._ Read the man page emerge(1) and ebuild(5) for details

**Comment by hirocaster** [comment]
Thanks, great Infomation. I will check it.

**Pull request merged**